native/exe/HCWIRBlasterCOM/dlldata.c

```diff
@@ -0,0 +1,68 @@
+/*********************************************************
+   DllData file -- generated by MIDL compiler 
+
+        DO NOT ALTER THIS FILE
+
+   This file is regenerated by MIDL on every IDL file compile.
+
+   To completely reconstruct this file, delete it and rerun MIDL
+   on all the IDL files in this DLL, specifying this file for the
+   /dlldata command line option
+
+*********************************************************/
+
+#define PROXY_DELEGATION
+
+#include <rpcproxy.h>
+
+#ifdef __cplusplus
+extern "C"   {
+#endif
+
+EXTERN_PROXY_FILE( HCWIRBlasterCOM )
+
+
+PROXYFILE_LIST_START
+/* Start of list */
+  REFERENCE_PROXY_FILE( HCWIRBlasterCOM ),
+/* End of list */
+PROXYFILE_LIST_END
+
+
+DLLDATA_ROUTINES( aProxyFileList, GET_DLL_CLSID )
+
+#ifdef __cplusplus
+}  /*extern "C" */
+#endif
+
+/* end of generated dlldata file */
+files in this DLL, specifying this file for the
```

Comment:
This file looks corrupted...like it's mostly duplicated.

Reply:
Weird.  That file is auto-generated by VS but must have gotten broken somehow.  It's like that in my local copy.  Perhaps it gets rebuilt during compile?  I'll fix it.